Correct return values for prompt dialog (#902)
MetaMask/metamask-docs: snaps/reference/rpc-api.md

##### Returns

> The text entered by the user if the prompt was submitted or `null` if the prompt was rejected or closed. If the user does not enter any text and submits the prompt, the value will be an empty string.

Reviewer: ```suggestion
The text entered by the user if the prompt was submitted or `null` if the prompt was rejected or closed. If the user does not enter any text and submits the prompt, the value is an empty string.
```
